Scuti/Scripts/ScutiSDK/ScutiUtils.cs

```diff
@@ -120,6 +120,49 @@ internal static string GetSupportEmail()
 #endif
 
     }
+
+    internal static bool HasLink(Offer offer)
```

Comment:
This may need to be renamed. Sounds like it is just checking for the link rather than opening the link.

Maybe TryOpenLink?